lib/puppet/provider/archive/wget.rb

```diff
@@ -0,0 +1,42 @@
+Puppet::Type.type(:archive).provide(:wget, :parent => :ruby ) do
+
+  commands   :wget => '/usr/bin/wget'
```

Comment:
Unless we must use /usr/bin/wget, it's better just to use wget and let puppet find the command in the path.